.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -32,21 +32,21 @@ body:
 
   - type: textarea
     attributes:
-      label: Describe the solution you'd like
+      label: Describe the solution you would like.
       placeholder: |
         A clear and concise description of what you want to happen.
     validations:
       required: true
 
   - type: textarea
     attributes:
-      label: Describe alternatives you've considered
+      label: Describe alternatives you have considered.
       description: If you have found a temporary fix, list it here.
     validations:
       required: true
 
   - type: textarea
     attributes:
-      label: Additional contex
+      label: Additional contex.
       description: Add any other context or screenshots about the feature request here.

```

Comment:
probably should be “context” not “contex”

Reply:
fixed